Gopkg.lock

```diff
@@ -631,6 +696,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "4de68f1342ba98a637ec8ca7496aeeae2021bf9e4c7c80db7924e14709151a62"
+  inputs-digest = "b643d3f62f346625b8be7e41b99e719b5d9d4c24f82141f1af5c01b20d2bac7c"
```

Comment:
can you remove this file from the PR? you must have run dep?

Reply:
@torkelo oh, sure, one moment